.github/workflows/pr_commit.yml

```diff
@@ -90,5 +90,7 @@ jobs:
           node-version: '16'
       - name: Install npm modules
         run: npm i
+      - name: Build plugins
+        run: npm run build
```

Comment:
Hi @sampullman 

I noticed that if I remove the `dist` folder on my machine, the same `spawn /bin/sh ENOENT` error would show up when running e2e test.
So I added a build command before e2e, which seems to fix the issue on my end, but not working in CI environment. 🤔

Reply:
That's odd, because I thought the E2E executor should automatically build the plugin. I'll take a look when I get the chance.